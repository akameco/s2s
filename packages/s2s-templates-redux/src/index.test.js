// @flow
import { templates } from '.'

test('snapshot', () => {
  expect(templates).toMatchSnapshot()
})
