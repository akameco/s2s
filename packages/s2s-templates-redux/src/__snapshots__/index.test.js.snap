// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Array [
  Object {
    "input": "containers.js",
    "test": /containers\\\\/\\.\\*\\\\/index\\.js/,
  },
  Object {
    "input": "components.js",
    "test": /components\\\\/\\.\\*\\\\/index\\.js/,
  },
  Object {
    "input": "component.test.js",
    "test": /components\\\\/\\.\\*\\\\/index\\.test\\.js/,
  },
  Object {
    "input": "reducer.js",
    "test": /reducer\\.js/,
  },
  Object {
    "input": "reducer.test.js",
    "output": "reducer.test.js",
    "test": /reducer\\.js/,
  },
  Object {
    "input": "actionTypes.js",
    "output": "actionTypes.js",
    "test": /reducer\\.js/,
  },
  Object {
    "input": "selectors.js",
    "test": /selectors\\.js/,
  },
  Object {
    "input": "selectors.test.js",
    "test": /selectors\\.test\\.js/,
  },
  Object {
    "input": "logic.js",
    "test": /logic\\.js/,
  },
]
`;
